app/components/Events.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+    <StackLayout>
+        <StackLayout class="eventItem" v-for="e in events" >
+            <Label class="category" :text="e.category" width="100%"/>
+            <Label class="title" :text="e.title" width="100%"/>
+            <Label class="text description" :text="e.description" width="100%"/>
+            <Label class="spacer" />
```

Comment:
do we need a class here?

Reply:
I left it in to give us just in case in the future we decide to customize spacing between the different lines of text we'd be able to change them easily with css.